### ISSUE TITLE
Cannot launch appium inspector- XCTRunner -  assertion failed: and Enqueue Failure: UI Testing Failure - App accessibility isn't loaded 

### ISSUE THREAD

**Issue by Sandarekha** [appium/appium#7959]
## The problem

Cannot launch appium inspector. It worked fine before. But now open app and close. Appium popup says "Could not start a new session."

## Environment

* Appium version - 1.6.3
* Desktop OS/version used to run Appium: Mac sierra 10.12
* Node.js version (unless using Appium.app|exe): v6.9.5
* Mobile platform/version under test: Xcode 8.2.1
* Real device or emulator/simulator: iPhone 7

## Details
App path and bundle ID has given correctly. 
Server start with >> ./node_modules/.bin/appium
But giving errors in log. 
assertion failed , Enqueue Failure: UI Testing Failure - App accessibility isn't loaded 

## Link to Appium logs
https://gist.github.com/Sandarekha/dbb5cbfaa745e48362b5f08a013ee9fc

**Comment by mykola-mokhnach** [comment]
This usually happens if the app cannot be started. Are you sure you did everything according to the [tutorial](https://github.com/appium/appium-xcuitest-driver/blob/master/docs/real-device-config.md)?

**Comment by Sandarekha** [comment]
Yep. I had followed it and it worked fine. I inspected 3 apps with it. But now appium give this error when I try to launch inspector.

**Comment by mykola-mokhnach** [comment]
It might be something is wrong with inspector settings. Can you make sure they are the same as for your automation framework (especially signing options)?

**Comment by Sandarekha** [comment]
Inspector settings are like this. 
What does it mean to check with automation framework? 

![screen shot 2017-03-02 at 12 59 08 pm](https://cloud.githubusercontent.com/assets/24934240/23497265/4b74b262-ff48-11e6-9070-4ba382ded1c6.png)
![screen shot 2017-03-02 at 12 59 22 pm](https://cloud.githubusercontent.com/assets/24934240/23497266/4d191bb2-ff48-11e6-9dd0-03b345bfea45.png)

**Comment by mykola-mokhnach** [comment]
Do not use this inspector. It is outdated and is not supported anymore.Try https://github.com/appium/appium-desktop/releases instead

**Comment by Sandarekha** [comment]
Thanks.. 
I'm working with it. Is there a way to rotate the screen view of the inspector?
My app is landscape but it does not fit the inspector view...

**Comment by silviu-geronimo** [comment]
Hello @Sandarekha could you please tell me how did you make it work?

I replaced the old Appium-desktop app with the new Appium 1.0.0 beta but I can't figure out how to link with a real iOS device. Is there any tutorial available?

**Comment by Sandarekha** [comment]
@silviu-geronimo  sorry for the late reply.
I deleted old version and install new one, and I tried with simulator only.
But can you try with adding capabilities device name and UDID?

**Comment by silviu-geronimo** [comment]
Could you please write here your capabilities items? I tried with these and the simulator it's opening, my applications and Web driver are installed but nothing is happening after. 

{
	"desiredCapabilities": {
		"platformName":"iOS",
		"newCommandTimeout":"7200",
		"app":"/Users/user/Downloads/App.ipa",
		"deviceName":"iPhone 6",
		"automationName":"XCUITest"
	}
}

**Comment by Sandarekha** [comment]
Sure..
It is like this. 
I have used .app

{
  "platformName": "iOS",
  "platformVersion": "10.2",
  "deviceName": "iPhone 7",
  "app": "/Users/admin/Library/Developer/Xcode/DerivedData/DD/bahgjrslauomylhhrtiunqloquog/Build/Products/Debug-iphonesimulator/abcd.app",
  "orientation": "LANDSCAPE",
  "autoAcceptAlerts": true
}
